prac_07/squaring.kv

```diff
@@ -15,11 +15,12 @@ BoxLayout:
         Label:
             id: output_label
             font_size: 48
+            color: (1,255,1,1)
 
     BoxLayout:
         orientation: 'vertical'
         multiline: False
```

Comment:
This declaration of BoxLayout is not necessary. you could just indent and use the one at the top instead